main.go

```diff
@@ -233,57 +231,48 @@ func run(state overseer.State) {
 	}
 
 	// Build include and exclude detector sets for filtering on engine initialization.
+	// Exit if there was an error to inform the user of the misconfiguration.
 	var includeDetectorSet, excludeDetectorSet map[config.DetectorID]struct{}
+	var customVerifierEndpoints map[config.DetectorID][]string
 	{
 		includeList, err := config.ParseDetectors(*includeDetectors)
 		if err != nil {
-			// Exit if there was an error to inform the user of the misconfiguration.
 			logFatal(err, "invalid include list detector configuration")
 		}
 		excludeList, err := config.ParseDetectors(*excludeDetectors)
 		if err != nil {
-			// Exit if there was an error to inform the user of the misconfiguration.
 			logFatal(err, "invalid exclude list detector configuration")
 		}
+		customVerifierEndpoints, err = config.ParseVerifierEndpoints(*verifiers)
```

Comment:
i know that `customVerifierEndpoints` is already verbose, but it took me a bit to figure out that it holds another set of detectors. can i talk you into renaming it to something like `detectorsWithCustomVerifierEndpoints`?